### PR TITLE
feat: add pr check redirects action

### LINKED ISSUE #62: add pr-check-redirects action
Currently, the `content` and `translated-content` both use the same action to check redirects. Improve maintenance by adding it here as a reusable workflow.

### PR DESCRIPTION
**Pull request by schalkneethling**
- [feat: add pr check redirects workflow](https://github.com/mdn/workflows/commit/4f62e02aa84c3708b19692a4ca7423df982bdd0e)
- [docs: for pre-check-redirect workflow](https://github.com/mdn/workflows/commit/fc073067fc15cd7b78f0c0f3d1705d4848b2f728)

Fix #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -738,3 +738,91 @@ jobs:
...
+
+Specify the target repository this action should run on. This is used to prevent actions from running on repositories other than the target repository. For example, specifying a `target-repo` of `mdn/workflows` will prevent the action from running on `fork/workflows`.
+
+- This `input` is required
+
+### Optional inputs
+
+#### cache
+
+Which package manager cache to use.
```
Comment (dipikabh) [inline README.md:765]:
```suggestion
The package manager cache to use.
```

File: README.md [hunk 2]
```diff
@@ -738,3 +738,91 @@ jobs:
...
+
+The node version to setup and use.
+
+- This `input` is optional, with a default of 16.
+
+#### target-locale
+
+The target locale for which to check the redirects file(s).
+
+- This `input` is optional, with a default of all locales in the repository.
```
Comment (dipikabh) [inline README.md:779]:
```suggestion
- This `input` is optional. The default is all locales in the repository.
```

File: README.md [hunk 3]
```diff
@@ -738,3 +738,91 @@ jobs:
...
+```yml
+name: check-redirects
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - files/**
+      - .github/workflows/pr-check_redirects.yml
```
Comment (dipikabh) [inline README.md:796]:
```suggestion
      - .github/workflows/pr-check-redirects.yml
```

File: README.md [hunk 4]
```diff
@@ -738,3 +738,91 @@ jobs:
...
+
+jobs:
+  check-redirects:
+    uses: mdn/workflows/.github/workflows/pre-check-redirects.yml@main
```
Comment (dipikabh) [inline README.md:800]:
```suggestion
    uses: mdn/workflows/.github/workflows/pr-check-redirects.yml@main
```

**Comment by schalkneethling** [comment]
> Spotted a few things that might need to be updated (such as, pr vs pre). I guess someone will also need to do a code review?

The code is all good as it is a copy of a workflow already in use. Thanks so much for fixing all my typos!

**Review by dipikabh — APPROVED** [review]
🎉

**Pull request merged**